promotion/datastore.go

```diff
@@ -860,6 +866,46 @@ func (pg *Postgres) GetOrder(orderID uuid.UUID) (*Order, error) {
 	return &order, nil
 }
 
+// EnqueueMintDrainJob - enqueue a mint drain job in "pending" status
+func (pg *Postgres) EnqueueMintDrainJob(ctx context.Context, walletID uuid.UUID, promotionIDs ...uuid.UUID) error {
+	tx, err := pg.RawDB().Beginx()
+	if err != nil {
+		return err
+	}
+	defer pg.RollbackTx(tx)
+
+	var mintDrainJob = MintDrainJob{}
+
+	statement := `
+	insert into mint_drain (wallet_id, total)
+	values ($1, (
+		select sum(approximate_value) from claims where wallet_id=$1 and promotion_id = any($2)
```

Comment:
According to https://github.com/brave-intl/bat-go/pull/611/files#diff-099010dcec7ac18795f69fe887fd07619ee78e12f09875283741fc7457cda235R1183, we use this total when determining the value of the resulting grant that is minted.

I don't think this is the right value, this is the total granted originally - rather than the amount of tokens that the user redeemed.

Reply:
I see what you mean.  Thanks.  In https://github.com/brave-intl/bat-go/pull/611/commits/2cf3fdd281227c566d0e0c91b37f78646396bcb2 I have adjusted to set the total per promotion in redeem and transfer after the redemption is completed.